Installation fails when installing ember-popper with npm

**Repeat Bug**
`$ ember install ember-attacher`

**Workaround**
`$ yarn add --dev ember-attacher`

```
$ ember install ember-attacher
NPM: Installed ember-attacher
installing ember-attacher
  install addon kybishop/ember-popper@^0.0.3
Command failed: npm install --save-dev --loglevel error kybishop/ember-popper@^0.0.3
npm ERR! addLocal Could not install /home/swerner/development/startup/frontend/kybishop/ember-popper@^0.0.3
npm ERR! path /home/swerner/development/startup/frontend/kybishop/ember-popper@^0.0.3
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open
npm ERR! enoent ENOENT: no such file or directory, open '/home/swerner/development/startup/frontend/kybishop/ember-popper@^0.0.3'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/swerner/.npm/_logs/2017-05-15T16_26_41_999Z-debug.log

Error: Command failed: npm install --save-dev --loglevel error kybishop/ember-popper@^0.0.3
npm ERR! addLocal Could not install /home/swerner/development/startup/frontend/kybishop/ember-popper@^0.0.3
npm ERR! path /home/swerner/development/startup/frontend/kybishop/ember-popper@^0.0.3
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open
npm ERR! enoent ENOENT: no such file or directory, open '/home/swerner/development/startup/frontend/kybishop/ember-popper@^0.0.3'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/swerner/.npm/_logs/2017-05-15T16_26_41_999Z-debug.log

    at Promise.all.then.arr (/home/swerner/development/startup/frontend/node_modules/execa/index.js:210:11)
    at process._tickCallback (internal/process/next_tick.js:109:7)
```
npm: 4.5.0
ember-source: 2.13.0

[2017-05-15T16_26_41_999Z-debug.txt](https://github.com/kybishop/ember-attacher/files/1001720/2017-05-15T16_26_41_999Z-debug.txt)

--- a/blueprints/ember-attacher/index.js
+++ b/blueprints/ember-attacher/index.js
@@ -4,7 +4,7 @@ module.exports = {
 
   afterInstall(options) {
     return this.addAddonToProject({
-      name: 'kybishop/ember-popper',
+      name: 'ember-popper',
       target: '^0.0.3',
       blueprintOptions: {
         save: options.save

